datahub-web-react/src/app/entity/shared/tabs/Documentation/components/editor/Editor.tsx

```diff
@@ -50,26 +50,26 @@ export const Editor = forwardRef((props: EditorProps, ref) => {
     const { manager, state, getContext } = useRemirror({
         extensions: () => [
             new BlockquoteExtension(),
-            new BoldExtension(),
-            new BulletListExtension(),
+            new BoldExtension({}),
```

Comment:
Even in the Remirror docs, they are not using an empty in constructor for these extensions: https://remirror.io/docs/extensions/bold-extension

Reply:
Here is the link of an online editor, using the same version of packages. Showing the error:

https://stackblitz.com/edit/vitejs-vite-z3rg1l?file=src%2FEditor.tsx
(You need to wait till yarn install is complete, then the tsx error will start showing)

---

![image](https://github.com/user-attachments/assets/f5b28e16-aad8-4a4f-920b-3722938cf67b)